lighthouse-core/lib/i18n/locales/ca.json

```diff
@@ -2060,6 +2069,9 @@
   "lighthouse-core/report/html/renderer/util.js | runtimeUnknown": {
     "message": "Desconegut"
   },
+  "lighthouse-core/report/html/renderer/util.js | showRelevantAudits": {
+    "message": "Show audits relevant to:"
```

Comment:
same...

prob tons more, should we have a test that the non-en strs don't === the en strs?